beacon-chain/blockchain/service.go

```diff
@@ -81,6 +81,10 @@ func (c *ChainService) Start() {
 		go c.blockProcessing()
 	} else {
 		log.Info("Waiting for ChainStart log from the Validator Deposit Contract to start the beacon chain...")
+		if c.web3Service == nil {
```

Comment:
Before running this check we need to check if `c.enablePOWChain` is true, otherwise the client will fail in the case where we didnt originally intend to use an actual POW chain